packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -1510,9 +1540,6 @@ describe('e2e', function () {
 
       describe('history file', function () {
         it('persists between sessions', async function () {
-          if (process.arch === 's390x') {
-            return this.skip(); // https://jira.mongodb.org/browse/MONGOSH-746
-          }
```

Comment:
Drive-by because this is also fixed by the env var fixup in .evergreen/setup-env.sh